src/Symfony/Bridge/Twig/Form/TwigRendererEngine.php

```diff
@@ -40,7 +40,7 @@ public function __construct(array $defaultThemes, Environment $environment)
     /**
      * {@inheritdoc}
      */
-    public function renderBlock(FormView $view, $resource, string $blockName, array $variables = [])
```

Comment:
Why do we remove the parameter type here?

Reply:
just noticed, fixing :)
i was fixing a test and messed up here :(